readme.md

```diff
@@ -2,45 +2,66 @@
 
 [<img src="https://babeljs.io/images/logo.svg" width="160" align="right" alt="babel">](http://babel.io)
 
-> A list of awesome Babel plugins, presets, etc.
+> A list of awesome Babel plugins, presets, etc. Many of these are from the community, but some are lesser-known
+> plugins in the Babel organization that may be useful to you.
 
 If you want to contribute, please read the [contribution guidelines](contributing.md).
 
-## Table of Contents
-
-- [Parsers](#parsers)
-- [Plugins](#plugins)
- - [Frameworks and Libraries](#frameworks-and-libraries)
- - [Testing](#testing)
-- [Presets](#presets)
-
 ## Parsers
 
  - [babel-eslint](https://github.com/babel/babel-eslint) - ESLint using Babel as the parser.
+   - Note: ESLint now lints most ES6+ syntax. This parser is only necessary if you are using Flow types or other experimental features.
 
 ## Plugins
 
-### Frameworks and Libraries
+### General Plugins
 
- - [React Optimize](https://github.com/thejameskyle/babel-react-optimize) - A Babel preset and plugins for optimizing React code.
- - [React Transform](https://github.com/gaearon/babel-plugin-react-transform) - Babel plugin to instrument React components with custom transforms
- - [c-3po](https://c-3po.js.org) - javascript localization tool based on es6 template literals which is implemented as a babel plugin.
- - [module-resolver](https://www.npmjs.com/package/babel-plugin-module-resolver) - Custom module resolver
- - [lodash](https://www.npmjs.com/package/babel-plugin-lodash) - Cherry-picks Lodash modules so you don’t have to.
- - [transform-react-remove-prop-types](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types) - Remove unnecessary React propTypes from the production build.
- - [react-require](https://www.npmjs.com/package/babel-plugin-react-require) - adds React import declaration if file contains JSX tags.
  - [emojification](https://www.npmjs.com/package/babel-plugin-emojification) - Turn all binding identifiers to unicode escaped emoji :smirk_cat:
```

Comment:
Honestly, this should be a high-quality, curated list, so we should probably just remove this one.